nixos/modules/services/web-apps/hitide.nix

```diff
@@ -0,0 +1,53 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+  cfg = config.services.hitide;
+in
+{
+  options.services.hitide = {
+    enable = mkEnableOption "the hitide service";
+
+    package = mkOption {
+      default = pkgs.hitide;
+      defaultText = "pkgs.hitide";
+      type = types.package;
+      description = "hitide package to use.";
+    };
+
+    backendHost = mkOption {
+      default = null;
+      type = types.string;
+      description = "URL path to lotide";
+      example = "http://localhost:3333/";
+    };
+
+    port = mkOption {
+      default = 4333;
+      type = types.int;
+      description = "Port number to bind to";
+    };
+  };
+
+  config = mkIf cfg.enable {
+    systemd.services.hitide = {
```

Comment:
Would you consider submitting a change upstream to include a `systemd` unit in their repo? If so, we could eventually (mostly) drop the unit from nixpkgs an utilize `systemd.packages = [ pkgs.hitide ];`. If you're up for that full bonus points will be awarded :tada:

Reply:
Not sure whether they would accept such patches upstream, but I will see what I can do and then report back here.

Reply:
Fantastic! Thanks.